### PR TITLE
FF115 Relnote: Sec-Purpose HTTP header

### PR DESCRIPTION
**Pull request by hamishwillee**
FF115 adds the `Sec-Purpose` HTTP header in https://bugzilla.mozilla.org/show_bug.cgi?id=1836328. This adds the release note.

Related docs work can be tracked in #27171

### CODE REVIEW

**Comment by bsmth** [comment]
Taking a look shortly 👀

File: files/en-us/mozilla/firefox/releases/115/index.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ This article provides information about the changes in Firefox 115 that affect d
 
 ### HTTP
 
+- The [`Sec-Purpose`](/en-US/docs/Web/HTTP/Headers/Sec-Purpose) HTTP {{Glossary("Fetch metadata request header", "fetch metadata request header")}} is now included in requests to {{Glossary("Prefetch")}} resources.
+  This allows servers to provide any special handling that might be needed, such as adjusting the caching expiry for the request.
+  ([Firefox bug 1836328](https://bugzil.la/1836328)).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/115/index.md:42]: Formatting nit:

```suggestion
  This allows servers to provide any special handling that might be needed, such as adjusting the caching expiry for the request ([Firefox bug 1836328](https://bugzil.la/1836328)).
```

File: files/en-us/mozilla/firefox/releases/115/index.md [hunk 2]
```diff
@@ -37,6 +37,10 @@ This article provides information about the changes in Firefox 115 that affect d
 
 ### HTTP
 
+- The [`Sec-Purpose`](/en-US/docs/Web/HTTP/Headers/Sec-Purpose) HTTP {{Glossary("Fetch metadata request header", "fetch metadata request header")}} is now included in requests to {{Glossary("Prefetch")}} resources.
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/115/index.md:40]: I presume you know, but this page doesn't exist. Is it incoming?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-Purpose

**Review by bsmth — APPROVED** [review]
Thanks, Hamish, two comments for your consideration, ready to go otherwise 👍🏻

File: files/en-us/mozilla/firefox/releases/115/index.md (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/115/index.md:40]: Yes :-). Sorry, I should have clarified that. It has merged now.

File: files/en-us/mozilla/firefox/releases/115/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/115/index.md:42]: Thank you.

**Pull request merged**